### DISCUSSION TITLE
Photos taken in Qfield on tablet not being pushed back into the Qfield cloud

### DISCUSSION THREAD

**Discussion by Bvpraagh** [opengisch/QField#4272] [Q&A]
Hi,
I have recently upgraded my desktop (windows) QGIS from V 3.10.6 to 3.28.6
Upgraded Qfield to v 2.7.6 which I use on my Galaxy Tab Active Pro 4

Since doing this, I have not been able to push photos captured in Qfield to the cloud. They are on my tablet in the DCIM folder but only the files are coming across to the cloud. I can work around by using a cable to download that folder but easier and less time consuming when pushed to the cloud like it used to.

The attribute widget table set up is slightly differently  in the later Qfield upgrade but I think I am setting up the widgets correctly (relative path?)
Everything worked fine in previous versions I was using.
Any suggestions on what I might be doing wrong?

![image](https://user-images.githubusercontent.com/104828477/236653547-59f49ad8-01da-4b9e-8e18-654ceca77282.png)

![image](https://user-images.githubusercontent.com/104828477/236653557-5ca6f3a5-2be5-4212-8807-a5ae5d909ea7.png)

**Comment by McGeo22** [comment]
Hi, I am definitly late but...
You have spacing in the photos' path/naming -> this could be a problem.
Have you check the size of the photos your device make? 
Have you try to include your "fid" field to the layout? 
_(the last suggestion is made based on my experience and I am not a QField developper so it might be non-sens...)_

**Comment by Bvpraagh** [comment]
Thanks very much for your reply. It was actually a space issue and I didn’t have enough file space left on the cloud!
